app/views/layouts/application.html.erb

```diff
@@ -7,8 +7,12 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'https://maps.googleapis.com/maps/api/js?key=AIzaSyAq7YSMMp-zWSRYBhmX59kFXb7yrw_aaOE' %>
+   <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+   <%= yield(:head_tags) %>
   </head>
```

Comment:
Simillarly, with turbolinks disabled, the `data-turbolinks-track` portion can be removed